### PR TITLE
Revise English

### PR DESCRIPTION
**Pull request by rosano**
I want to look over all the English strings. Here are just a few, should we try merging a small amount to see what happens in Transifex (in terms of causing / minimizing notifications for other languages) first? Or should I go through the whole file?

### CODE REVIEW

**Comment by rosano** [comment]
@r10s I reverted the capitalization. Should I continue revising the rest of the file or would you prefer just doing a small change?

File: res/values/strings.xml [hunk 1]
```diff
@@ -33,13 +33,13 @@
...
-    <!-- In the meaning of "To archive", "Put a chat to the archive". This string does not mean "The Archive". -->
+    <!-- Verb "to archive", as in "put a chat in the archive", not a noun "The Archive". -->
     <string name="archive">Archive</string>
-    <!-- In the meaning of "To unarchive", "Remove a chat from the archive", opposite of the previous string -->
+    <!-- Verb "to unarchive", as in "remove a chat from the archive", opposite of the previous string -->
     <string name="unarchive">Unarchive</string>
     <string name="mute">Mute</string>
     <string name="ephemeral_messages">Disappearing Messages</string>
-    <string name="ephemeral_messages_hint">These settings apply to all chat members using Delta Chat. However, they may copy, save and forward messages or use other e-mail clients.</string>
+    <string name="ephemeral_messages_hint">Removes messages after the specified duration from all members of this chat; this applies only if they use Delta Chat and does not prevent copying, saving, and forwarding messages or using other e-mail clients.</string>
```
Comment (r10s) [inline res/values/strings.xml:42]: tbh,i like the old sentence more. there was quite some effort put into make it that short and precise (place is limited!) 

eg. it is pretty clear by the context that it is about "removing messages after some duration".

Comment (rosano) [reply res/values/strings.xml]: Now one character shorter than the original

**Comment by r10s** [comment]
> @r10s I reverted the capitalization as per https://github.com/deltachat/deltachat-android/pull/1979. Should I continue revising the rest of the file or would you prefer just doing a small change?

i think, we figured out the scope of the possible changes, you can continue in this pr if you like. larger rewordings, making things much longer etc. should go to separate prs.

**Comment by rosano** [comment]
In what context does this string appear?
https://github.com/deltachat/deltachat-android/blob/7cc844af95589ad8586730a386c7f304de048d0c/res/values/strings.xml#L290

**Comment by r10s** [comment]
> In what context does this string appear?
> `<string name="videochat_you_invited_hint">You invited to a video chat.</string>`

it is shown as a message on your device when you send a videochat invitation (you need to enable videochat at "experimental features" for this). 
it is shown in the message that says "Tap to Open"

File: res/values/strings.xml [hunk 2]
```diff
@@ -285,24 +285,24 @@
 
     <string name="videochat">Video Chat</string>
     <string name="videochat_invite_user_to_videochat">Invite %1$s to a video chat?</string>
-    <string name="videochat_invite_user_hint">This requires a compatible app or a compatible browser on both ends.</string>
+    <string name="videochat_invite_user_hint">This requires a compatible app or browser on both ends.</string>
     <string name="videochat_contact_invited_hint">%1$s invited to a video chat.</string>
-    <string name="videochat_you_invited_hint">You invited to a video chat.</string>
+    <string name="videochat_you_invited_hint">You were invited to a video chat.</string>
```
Comment (adbenitez) [inline res/values/strings.xml:290]: mmmh... I think this is not right, the string is shown to yourself like "you did invite to a video chat" on the receiver side it is "Foo invited to a video chat" look the previous string

**Comment by r10s** [comment]
i also added @link2xt for review as many of the very precise wordings come from him :)

for me, this looks good all in all :)

**Comment by rosano** [comment]
When does this appear? It reads to me as 'The group left' but maybe it means 'You left the group'?

https://github.com/deltachat/deltachat-android/blob/7cc844af95589ad8586730a386c7f304de048d0c/res/values/strings.xml#L714

**Comment by r10s** [comment]
"Group left." is shown as the (hopefully :) last message when the user left a group using the menu command "Leave Group".

**Comment by rosano** [comment]
Ok I'm finished and will leave it for review.

File: res/values/strings.xml [hunk 3]
```diff
@@ -617,7 +617,7 @@
     <string name="pref_watch_sent_folder">Watch Sent Folder</string>
     <string name="pref_send_copy_to_self">Send Copy to Self</string>
     <string name="pref_send_copy_to_self_explain">Required when using this account on multiple devices.</string>
-    <string name="pref_auto_folder_moves">Automatic Moves to DeltaChat Folder</string>
+    <string name="pref_auto_folder_moves">Move automatically to DeltaChat Folder</string>
```
Comment (r10s) [inline res/values/strings.xml:620]: the old one looks a bit better to me in the context of the "IMAP Folder Handling" - or is the old one just wrong english?

this is not a blocker, however.

**Review by r10s — APPROVED** [review]
i have some minor comments, but all in all this looks very good!

also nice that most strings are shorter now :)

File: res/values/strings.xml (hunk 3 above)
Comment (rosano) [reply res/values/strings.xml:620]: The old one wasn't wrong but a bit more awkward and opaque, there's no ambiguity about the second one.

File: res/values/strings.xml [hunk 4]
```diff
@@ -250,16 +250,16 @@
     <string name="title_share_location">Share location with all group members</string>
     <string name="device_talk">Device Messages</string>
     <string name="device_talk_subtitle">Locally generated messages</string>
-    <string name="device_talk_explain">Messages in this chat are generated locally by your Delta Chat app. Its makers use it to inform about app updates and problems during usage.</string>
+    <string name="device_talk_explain">Messages in this chat are generated on your device and come from the makers of Delta Chat to inform about app updates and problems during usage.</string>
```
[the remark this answers is not in the document]
Comment (rosano) [reply res/values/strings.xml]: Maybe like this? Feel free to suggest an alternative: https://github.com/deltachat/deltachat-android/blob/98d7efef677c29b74dc72937b4d76515430a64ce/res/values/strings.xml#L253

File: res/values/strings.xml [hunk 5]
```diff
@@ -315,17 +315,17 @@
...
-    <string name="ask_unblock_contact">Unblock this contact? You will once again be able to receive messages from this contact.</string>
-    <string name="ask_delete_contacts">Delete contacts?\n\nContacts with ongoing chats and contacts from the system\'s address book cannot be deleted permanently.</string>
-    <string name="ask_delete_contact">Delete contact %1$s?\n\nContacts with ongoing chats and contacts from the system\'s address book cannot be deleted permanently.</string>
+    <string name="ask_export_attachment">Exporting attachments will allow other apps on your device to access them.\n\nContinue?</string>
+    <string name="ask_block_contact">Block this contact? You will no longer receive messages from them.</string>
+    <string name="ask_unblock_contact">Unblock this contact? You will then be able to receive messages from them.</string>
+    <string name="ask_delete_contacts">Delete contacts?\n\nContacts with ongoing chats and from the system address book cannot be deleted permanently.</string>
+    <string name="ask_delete_contact">Delete contact %1$s?\n\nContacts with ongoing chats and contacts from the system address book cannot be deleted permanently.</string>
```
[the remark this answers is not in the document]
Comment (rosano) [reply res/values/strings.xml:322]: Could you clarify what was removed? I don't follow. And what is the other string you refer to?

**Comment by rosano** [comment]
I'm not sure how merging works here with Transifex, does it automatically go there or does one have to do something specific?

File: res/values/strings.xml (hunk 5 above)
[the remark this answers is not in the document]
Comment (r10s) [reply res/values/strings.xml:322]: there is now an imbalance between `ask_delete_contacts` and `ask_delete_contact` - one says "Contacts with ongoing chats and from the system address book" the other says "Contacts with ongoing chats and _contacts_ from the system address book".

to fix this, either also remoe "contacts" in `ask_delete_contact`:

```suggestion
    <string name="ask_delete_contact">Delete contact %1$s?\n\nContacts with ongoing chats and the system address book cannot be deleted permanently.</string>
```

or re-add it to the other string. while the latter is shorter, maybe the old one is clearer.

i first read that it is about deleting the system address book, tbh.

so, maybe the old string with the repetition is better and clearer, though a bit longer and maybe not that polished.

but i personally would still be fine with both variants, but there should be no imbalance, see above.

**Comment by r10s** [comment]
> I'm not sure how merging works here with Transifex, does it automatically go there or does one have to do something specific? I'm inclined to ask someone else to do that.

i can merge this; afterwards, i will immediately pull all translations from transifex and push everything again with the change translation source (english). this step is a bit critical, it is easy to loose translations just because of a changed comma in the source :)

File: res/values/strings.xml (hunk 5 above)
[the remark this answers is not in the document]
Comment (r10s) [reply res/values/strings.xml:322]: for the short variant: isn't an "or" better?

"Contacts with ongoing chats or from the system address book cannot be deleted permanently."

**Review by rosano — COMMENTED** [review]
Ok I think everything has been resolved. I went with 'or' before I saw the repeating 'contacts' suggestion. Feel free to change it. Ready to merge.

**Comment by r10s** [comment]
great to have that happen!

after merging, i ran `./scripts/tx-update-changed-sources.sh` so that the changes sources and help texts are available on transifex, without deleting the existing translations.

**Pull request merged**